src/model/modeling/adapter.py

```diff
@@ -100,10 +101,12 @@ def apply_medium(model, medium):
     medium_mapping = {}
     for compound in medium:
         try:
-            metabolite = find_metabolite(model, compound.id, compound.namespace, 'e')
+            extracellular = find_external_compartment(model)
```

Comment:
Calling this function may raise a `RuntimeError` which should be handled. I suggest that if the model's external compartment cannot be identified we point them to memote.io or ask them to contact us directly. Definitely also want to be notified on Sentry.

Reply:
Good call. Any chance to change it to a bit more specific exception in cobrapy?

Reply:
Yes, exceptions in cobrapy are a bit of a mess at the moment but something we should be able to do. Do you want to make a suggestion?